bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/niri/InstNIRI.java

```diff
@@ -263,13 +270,43 @@ public double getSetupTime(ISPObservation obs) {
         return (altair == null) ? getSetupTime(mode) : getSetupTime(mode, altair.getMode());
     }
 
+    public double getSetupTime(Config[] conf) {
+        String aoSystem = (String) conf[0].getItemValue(AOConstants.AO_SYSTEM_KEY);
```

Comment:
Why is it hardcoded to be `conf[0]`?

Reply:
conf[0] is used to determine which setup time to use for the sequence (setup time for some instruments can vary depending on the observing mode/components used). I'm assuming that the sequence always has at least one step; this first step has all the information needed to determine the type of setup. Now that you asked, I'm thinking I should add a check that there is actually at least one step in that sequence...